backfill/2021-09-07-error-geo/README.md

```diff
@@ -0,0 +1,125 @@
+# Reprocess from payload_bytes_raw due to incorrect IP lookup
+
+See https://bugzilla.mozilla.org/show_bug.cgi?id=1729069
+
+Last time we needed to reprocess geo was
+https://github.com/mozilla/bigquery-backfill/tree/main/backfill/2020-02-07-error-geo
+
+
+## Setup
+
+```
+gcloud auth application-default login
+```
+
+
+## Stub installer
+
+
+Let's do all of the `stub_installer` pipeline family backfill, since the data size is so small.
+
+First, we backfill from the standard PBR table for stub_installer:
+
+```
+PROJECT="moz-fx-data-backfill-6"
+JOB_NAME="geo-backfill"
+PIPELINE_FAMILY="stub_installer"
+
+## this script assumes it's being run from the ingestion-beam directory
+## of the gcp-ingestion repo.
+
+mvn clean compile exec:java -Dexec.mainClass=com.mozilla.telemetry.Decoder -Dexec.args="\
+    --runner=Dataflow \
+    --jobName=$JOB_NAME \
+    --project=$PROJECT \
+    --geoCityDatabase=gs://moz-fx-data-prod-geoip/GeoIP2-City/20210903/GeoIP2-City.mmdb \
+    --geoIspDatabase=gs://moz-fx-data-prod-geoip/GeoIP2-ISP/20210903/GeoIP2-ISP.mmdb \
+    --geoCityFilter=gs://moz-fx-data-prod-dataflow-templates/cities15000.txt \
+    --schemasLocation=gs://moz-fx-data-prod-dataflow/schemas/202109030341_302c86e7.tar.gz \
+    --inputType=bigquery_table \
+    --input=\"moz-fx-data-shared-prod:payload_bytes_raw.${PIPELINE_FAMILY}\" \
+    --bqReadMethod=storageapi \
+    --outputType=bigquery \
+    --bqRowRestriction=\"CAST(submission_timestamp AS DATE) BETWEEN '2021-08-20' AND '2021-09-03'\"
+    --bqWriteMethod=file_loads \
+    --bqClusteringFields=submission_timestamp \
+    --output=${PROJECT}:\${document_namespace}_live.\${document_type}_v\${document_version} \
+    --errorOutputType=bigquery \
+    --errorOutput=${PROJECT}:payload_bytes_error.${PIPELINE_FAMILY} \
+    --experiments=shuffle_mode=service \
+    --region=us-central1 \
```

Comment:
I don't think this is what caused the quota issue but we generally use us-west1 for workloads. If we had quotas bumped for these projects from e.g. the initial AWS import they may have only applied to us-west1.